src/components/AssetsTable/index.jsx

```diff
@@ -386,44 +405,19 @@ export class AssetsTable extends React.Component {
     );
   }
 
-  /**
-   * Rendering an empty status alert keeps the alert in the DOM initially and
-   * enables screen readers to listen for changes.
-   */
-  renderEmptyStatusAlert() {
-    return (
-      <div>
-        <StatusAlert
-          dialog=""
-          open={false}
-          dismissible={false}
-        />
-      </div>
-    );
-  }
-
   renderStatusAlert() {
     const { assetsStatus } = this.props;
-    const deleteActions = [assetActions.DELETE_ASSET_FAILURE, assetActions.DELETE_ASSET_SUCCESS];
-    const uploadActions = [assetActions.UPLOAD_ASSET_SUCCESS,
-      assetActions.UPLOAD_EXCEED_MAX_COUNT_ERROR, assetActions.UPLOAD_EXCEED_MAX_SIZE_ERROR,
-      assetActions.UPLOAD_ASSET_FAILURE, assetActions.UPLOADING_ASSETS];
-    let status = {};
+
     let onClose = this.closeStatusAlert;
-    if (assetsStatus.type === assetActions.TOGGLING_LOCK_ASSET_FAILURE) {
-      status = this.getLockFailedStatusFields(assetsStatus);
-    } else if (deleteActions.includes(assetsStatus.type)) {
-      status = this.getDeleteStatusFields(assetsStatus);
+    if (this.deleteActions.includes(assetsStatus.type)) {
```

Comment:
I was just looking at how Array.includes works, and I saw that it's [not supported](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) by Internet Explorer :(. It seems to me that babel has a [polyfill](https://github.com/zloirock/core-js#stage-4-proposals) in the babel-polyfill pacakage, so I think if you do `import "babel-polyfill";` at the top of the file, that should take care of it. I found that [here](https://babeljs.io/docs/usage/polyfill/). I think there might be some other places in the codebase where we use Array.includes, so we should polyfill it there. I'm not sure whether that's critical to do now, or whether we can file a ticket to do later. I know Internet Explorer isn't used super often, but I'd hate to release something and then have someone use Internet Explorer and see it break :/. I'd be interested in other people's opinions on that, though.

Reply:
If we go the route of separating out the `actionTypes` above to more distinct types we should be able to leverage `if ('key' in myObj)` for logical checks like this as we're be checking if it's in an `Object` instead of an `Array`. I think this kind of makes me want to push to organizing those action types more distinctly so that we don't need to use `includes`.